packages/expo-image-loader/ios/EXImageLoader/EXImageLoader.m

```diff
@@ -0,0 +1,38 @@
+#import <EXImageLoader/EXImageLoader.h>
+#import <React/RCTImageLoader.h>
+
+@interface EXImageLoader ()
+
+@property (weak, nonatomic) RCTBridge *bridge;
+
+@end
+
+@implementation EXImageLoader
+
+UM_REGISTER_MODULE();
+
++ (NSString *)moduleName
+{
+  return @"EXImageLoader";
+}
+
++ (const NSArray<Protocol *> *)exportedInterfaces
+{
+  return @[@protocol(UMImageLoaderInterface)];
+}
+
+- (void)setBridge:(RCTBridge *)bridge
+{
+  _bridge = bridge;
+}
+
+- (void)loadImageForURL:(NSURL *)imageURL
+      completionHandler:(UMImageLoaderCompletionBlock)completionHandler
+{
+   [_bridge.imageLoader loadImageWithURLRequest:[NSURLRequest requestWithURL:imageURL]
```

Comment:
Since we're touching this code, we could also fix [this deprecation issue](https://github.com/facebook/react-native/blob/30c1196d7da14dd91cbeeb1d495e9fec6a19ea19/React/CoreModules/RCTImageLoader.mm#L955-L956):

```objc
/**
 * DEPRECATED!! DO NOT USE
 * Instead use `[_bridge moduleForClass:[RCTImageLoader class]]`
 */
@implementation RCTBridge (RCTImageLoader)

- (RCTImageLoader *)imageLoader
{
  RCTLogWarn(@"Calling bridge.imageLoader is deprecated and will not work in newer versions of RN. Please update to the"
             "moduleForClass API or turboModuleLookupDelegate API.");
  return [self moduleForClass:[RCTImageLoader class]];
}
// ...
```